types/chartmogul-node/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "private": true,
     "name": "@types/chartmogul-node",
-    "version": "2.0.9999",
```

Comment:
I think this is the `@types/chartmogul-node` version this repo will publish, not our `chartmogul-node` SDK version. And [the convention](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/e7d5d04911a66380417a1427d683ad6421df1697/README.md#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library) seems to be major+minor should match our SDK major+minor version, so if you plan to publish our SDK version `3.8.0` then this version should be `3.8.9999`. At least that's how I understand it.